pkg/serverless/aws/arn.go

```diff
@@ -224,6 +233,18 @@ func GetARNTags() []string {
 	return tags
 }
 
+// BuildGlobalTagsMap returns tags associated with the given ARN
+func BuildGlobalTagsMap(functionARN string, functionName string, region string, awsAccountID string) map[string]string {
```

Comment:
If we do move this file to `arn.go` (which Darcy and I disagree about) can we rename it so that it is clear that this is tags for tracing? This function is not used for logs/metrics if I am not mistaken. (And to reiterate my original suggestion, I don't think we should put this tracing-specific function in `arn.go` 🙂 )